test/integration/FileBackendEncodedSlashHandling.test.ts

```diff
@@ -39,12 +39,11 @@ describe('A server with a file backend storage', (): void => {
   });
 
   afterAll(async(): Promise<void> => {
-    // Await removeFolder(rootFilePath);
```

Comment:
I missed this on the previous PR, but this line should be here (but not commented out of course).